File: KL_Segitiga.cpp
#include <iostream>
using namespace std;

int main()
{
    int menu;
    float a, b, c, alas, tinggi, keliling, luas;

    cout << "=============== KELILING & LUAS SEGITIGA ===============\n\n";
    cout << "1. Keliling Segitiga\n2. Luas Segitiga\n\n";
    cout << "Inputkan Nomor Menu            = "; cin >> menu;
    cout << endl;

    switch(menu){
        case 1 :
            cout << "Inputkan Sisi Pertama Segitiga = "; cin >> a;
            cout << "Inputkan Sisi Kedua Segitiga   = "; cin >> b;
            cout << "Inputkan Sisi Ketiga Segitiga  = "; cin >> c;
            keliling = a + b + c;
            cout << "\nKeliling Segitiga              = " << keliling << endl;
            break;
        case 2 :
            cout << "Inputkan Alas Segitiga         = "; cin >> alas;
            cout << "Inputkan Tinggi Segitiga       = "; cin >> tinggi;
            luas = 0.5 * (alas * tinggi);
            cout << "\nLuas Segitiga                  = " << luas << endl;
            break;
        default:
            cout << "Input Salah" << endl;
    }
}

File: KL_Segitiga.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author USER
 */
public class KL_Segitiga {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int menu;
        double a, b, c, alas, tinggi, keliling, luas;

        System.out.println("=============== KELILING & LUAS SEGITIGA ===============\n\n");
        System.out.println("1. Keliling Segitiga\n2. Luas Segitiga\n\n");
        System.out.print("Inputkan Nomor Menu            = ");
        menu = sc.nextInt();
        System.out.println("");

        switch (menu){
            case 1:
                System.out.print("Inputkan Sisi Pertama Segitiga = ");
                a = sc.nextDouble();
                System.out.print("Inputkan Sisi Kedua Segitiga   = ");
                b = sc.nextDouble();
                System.out.print("Inputkan Sisi Ketiga Segitiga  = ");
                c = sc.nextDouble();
                keliling = a + b + c;
                System.out.println("\nKeliling Segitiga              = " + keliling);
                break;
            case 2:
                System.out.print("Inputkan Alas Segitiga         = ");
                alas = sc.nextDouble();
                System.out.print("Inputkan Tinggi Segitiga       = ");
                tinggi = sc.nextDouble();
                luas = 0.5 * (alas * tinggi);
                System.out.println("\nLuas Segitiga                  = " + luas);
                break;
            default:
                System.out.println("Input Salah");
        }
    }
}
